torch/_dynamo/variables/builtin.py

```diff
@@ -660,6 +660,10 @@ def never(tx: "InstructionTranslator", a, b):
                             (VariableTracker, SymNodeVariable),
                             op_var._comparison_with_symnode,
                         ),
+                        (
+                            (variables.ExceptionVariable, variables.ExceptionVariable),
+                            lambda tx, l, r: ConstantVariable(op(l, r)),
```

Comment:
I'll test the case where we do the comparison only if the type of the exceptions raised are different. 

```
is(ValueError, ValueError) => Graph break
is(ValueError, RuntimeError) => False
is_not(ValueError, RuntimeError) => True
```